docs/models/blockstates/forgeBlockstates.md

```diff
@@ -0,0 +1,140 @@
+Forge's Blockstates
+===================
+
+Forge has its own blockstate json format to accommodate for modders needs. It introduces submodels, which allows you to build the final blockstate from different parts.
+
+!!! Attention
+
+    Note that all models and textures referenced are from vanilla minecraft. For your own mod, you have to use the full location! For example: `"mymod:blocks/blockTexture"`.
+
+    You don't have to use Forge's blockstate format at all, you can also use the vanilla format!
+
+General Structure of the Format
+-------------------------------
+
+```json
+{
+  "forge_marker": 1,
+  "defaults": {
+    "textures": {
+      "all": "blocks/dirt"
+    },
+    "model": "cube_all",
+    "uvlock": true
+  },
+  "variants": {
+    "normal": [{}]
+  }
+}
+```
+
+This json declares a simple blockstate that has dirt on each side. Let's go through it step by step.
+
+```json
+  "forge_marker": 1,
+```
+
+This tells the game that the blockstate json is the one from Forge, not from vanilla Minecraft.
+The 1 is the version of the format, which ensures that old blockstate JSONs can be supported should the format ever change. Currently there is only this one.
+
+```json
+  "defaults": {
+    "textures": {
+      "all": "blocks/dirt"
+    },
+    "model": "cube_all",
+    "uvlock": true
+  }
+```
+
+The defaults section contains the default values for all variants. They can be overwritten by the variants. The defaults section is **optional**! You do not need to define defaults, the block can be omitted altogether.
+
+```json
+  "variants": {
+    "normal": [{}]
```

Comment:
Could you please include the information from Jay Avery's very informative post [here](http://www.minecraftforge.net/forum/topic/57153-1102-issue-with-fluid-json-files/?do=findComment&comment=266664), regarding fully defined and partially defined variants.